app-text/poppler/poppler-21.12.0.recipe

```diff
@@ -255,10 +253,13 @@ PROVIDES="
 	cmd:pdftops
 	cmd:pdftotext
 	cmd:pdfunite
+	lib:libpoppler$secondaryArchSuffix = $libVersionCompat
+	lib:libpoppler_cpp$secondaryArchSuffix = $libCppVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	poppler_data >= 0.4.10
+	poppler_data >= 0.4.11
+#	nss${secondaryArchSuffix}_devel
```

Comment:
Why this comment here?

Reply:
Probably a left over trying to fix the issue, can be removed